From 1287d8d2fbc38bb0ab507756281e8d726cf92464 Mon Sep 17 00:00:00 2001
From: Guido Guenther <agx@sigxcpu.org>
Date: Wed, 5 Oct 2011 19:52:11 +0200
Subject: [PATCH] xc_domain_restore: reset the completed flag

otherwise long running processes like libvirt can resume a domain only
once.
---
 a/tools/libxc/xc_domain_restore.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/tools/libxc/xc_domain_restore.c b/tools/libxc/xc_domain_restore.c
index 083927a..6edd584 100644
--- a/tools/libxc/xc_domain_restore.c
+++ b/tools/libxc/xc_domain_restore.c
@@ -1145,6 +1145,7 @@ int xc_domain_restore(xc_interface *xch, int io_fd, uint32_t dom,
     memset(&tailbuf, 0, sizeof(tailbuf));
     tailbuf.ishvm = hvm;
 
+    ctx->completed = 0;
     /* For info only */
     ctx->nr_pfns = 0;
 
-- 
1.7.2.5

